tests/scripts/arp_responder.py

```diff
@@ -10,8 +10,8 @@
 import logging
 import ptf.packet as scapy
 import scapy.all as scapy2
-import scapy.arch.pcapdnet # noqa F401
 scapy2.conf.use_pcap = True
+import scapy.arch.pcapdnet # noqa F401
```

Comment:
Do you want to put a commented log here to maintain this import sequence?

Reply:
Sai's PR will overwrite this file 
https://github.com/sonic-net/sonic-mgmt/pull/8697/files